src/test/java/uk/nhs/prm/deduction/e2e/TestConfiguration.java

```diff
@@ -93,6 +93,15 @@ public String getEhrRepoE2EApiKey() {
         return awsConfigurationClient.getParamValue(String.format("/repo/%s/user-input/api-keys/ehr-repo/e2e-test", getEnvironmentName()));
     }
 
+    public String getEhrOutPostgresJdbcUrl() {
+        String hostname = awsConfigurationClient.getParamValue(String.format("/repo/%s/output/prm-repo-ehr-out-service/db-host", getEnvironmentName()));
+        String password = awsConfigurationClient.getParamValue(String.format("/repo/%s/user-input/ehr-out-service-db-password", getEnvironmentName()));
+        String username = awsConfigurationClient.getParamValue(String.format("/repo/%s/user-input/ehr-out-service-db-username", getEnvironmentName()));
+        String dbname = awsConfigurationClient.getParamValue(String.format("/repo/%s/output/prm-repo-ehr-out-service/db-name", getEnvironmentName()));
+        String port = "5432";
+        return "jdbc:postgresql://" + hostname + ":" + port + "/" + dbname + "?user=" + username + "&password=" + password;
```

Comment:
I would say the port probably doesn't need a variable of it's own!

```suggestion
        return "jdbc:postgresql://" + hostname + ":5432/"+ dbname + "?user=" + username + "&password=" + password;
```